src/README.md

```diff
@@ -0,0 +1,9 @@
+## Other Algs which we use (and thus won't implement ourselves)
```

Comment:
what's a better wording?

Reply:
I don't think what you have is bad, but here is an alternative since you asked for one

"Better implementations than we could write"

Anything we reference should actually be better than anything we could write otherwise we should just write it ourselves (better)